src/app/common/components/modals/dialog-create-or-update-wallet/dialog-create-or-update-wallet.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit, Inject } from '@angular/core';
+import {MatDialogRef, MAT_DIALOG_DATA} from '@angular/material/dialog';
+
+@Component({
+  selector: 'app-dialog-create-or-update-wallet',
+  templateUrl: './dialog-create-or-update-wallet.component.html',
+  styleUrls: ['./dialog-create-or-update-wallet.component.scss']
+})
+export class DialogCreateOrUpdateWalletComponent implements OnInit {
+
+    constructor(
+        public dialogRef: MatDialogRef<DialogCreateOrUpdateWalletComponent>,
+        @Inject(MAT_DIALOG_DATA) public data: { title: string, btn: string, inputData: string}
```

Comment:
Also please do not provide `btn` property. Instead lets maybe have property called `isNewWallet` or `isEditingWallet` and based on that you can show proper text for button in the template